.github/workflows/publish.yml

```diff
@@ -59,7 +59,19 @@ jobs:
         uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
         with:
           egress-policy: audit
-
+      # Checkout the code again for release
+      - name: Checkout
+        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+      - name: Use Node.js
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 20
+      # Run install dependencies
+      - name: Install dependencies
+        run: npm run install:all
+      # Ensure project builds successfully before creating release
+      - name: Build
```

Comment:
I don't think we should need the 'Build' step here. The failing step which creates the vsix must be running `vsce package`, which runs the `vscode:prepublish` script, which itself runs webpack.